modules/ROOT/pages/_partials/data-sync/server-realtime-updates-amq.adoc

```diff
@@ -184,33 +217,53 @@ const client = mqtt.connect({
 
 const pubsub = new MQTTPubSub({ client })
 ----
-
-There are some additional options passed into `mqtt.connect`
++
+[NOTE]
+====
```

Comment:
@darahayes  I think this applies to either method, so should prob move it to end of whole procedure?

Reply:
@finp these are additional options only needed to enable TLS connections specifically when dealing with an AMQ broker in OpenShift. This is why I had that under its own heading originally. In my eyes, it has enough content and additional information the user needs to be aware of to warrant its own section.

Reply:
I've moved it to it's own step, ok?